.xrefcheck.yaml

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2022 Tocqueville Group
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# Parameters of scanners for various file types.
+scanners:
+  markdown:
+    # Flavor of markdown, e.g. GitHub-flavor.
+    #
+    # This affects which anchors are generated for headers.
+    flavor: GitHub
```

Comment:
This file is not needed - `GitHub` is the default flavor

Reply:
deleted it